style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 10px;
+  font-family: sans-serif;
+  width: 100%;
+  max-width: 100%;
```

Comment:
If you set `width: 100%`, why do you also need `max-width: 100%` here?

Reply:
DONE!